build_scripts/windows/scripts/build.cmd

```diff
@@ -72,6 +87,9 @@ if not exist %WIX_DIR% (
     popd
 )
 
+REM Use only downloaded python and ignore machine state
+set PYTHONHOME=%PYTHON_DIR%
+
```

Comment:
```suggestion
```
Once `PYTHONHOME` is defined.  `azure-cli` will be installed in `PYTHON_DIR`(artifacts\x64\Python\Lib) instead of `BUILDING_DIR` (artifacts\x64\cli\Lib).

Reply:
I think I'm going to back out all the changes for rehoming the bits anyway. The idea was to make it easy to build packages for either arch on a single machine, but I'm worried about downstream effects in build, publishing, testing, etc. Already the plan was to have separate agents build. A local dev can simply clean the output folder, re-set env vars, and build again.

Reply:
I would recommend, though, considering that in a separate PR that fixes multi-target build. Most build systems support this, and in the long run you're probably going to want to if you keep building x86 bits.

Reply:
`PYTHON_DIR` (artifacts\x64\Python\Lib) is not in MSI package path, so these libraries are not in final MSI.
The build MSI is 15MB, which is much smaller than normal size (50MB).